### PR TITLE
fix(connector): [Tsys] Update endpoint and unit tests

### PR DESCRIPTION
**Pull request by SamraatBansal**
## Type of Change

- [X] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Update the case of the endpoint in payments and refunds flow to make it lowercase consistent.
Update Failing Unit Tests 

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context

Update the case of the endpoint in payments and refunds flow to make it lowercase consistent.
Update Failing Unit Tests 

## How did you test it?

**Tests**
Screen Shot 2023-07-17 at 6 09 21 PM

## Checklist

- [X] I formatted the code `cargo +nightly fmt --all`
- [X] I addressed lints thrown by `cargo clippy`
- [X] I reviewed submitted code
- [X] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: crates/router/src/connector/tsys/transformers.rs [hunk 1]
```diff
@@ -464,6 +464,7 @@ impl From<TsysTransactionDetails> for enums::RefundStatus {
     fn from(item: TsysTransactionDetails) -> Self {
         match item.transaction_status {
             TsysTransactionStatus::Approved => Self::Pending,
+            //Connector calls refunds as Void
             TsysTransactionStatus::Void => Self::Success,
```
Comment (pixincreate) [inline crates/router/src/connector/tsys/transformers.rs:468]: If refund is called as void, what do they call for voiding a payment? Or do they even support?

Comment (SamraatBansal) [reply crates/router/src/connector/tsys/transformers.rs:468]: Both Void and Refunds are addressed as `Void`
We handle them accordingly in response on the designated call either payments or refunds

File: crates/router/tests/connectors/tsys.rs [hunk 2]
```diff
@@ -286,9 +296,14 @@ async fn should_refund_succeeded_payment_multiple_times() {
 #[actix_web::test]
 async fn should_sync_refund() {
     let refund_response = CONNECTOR
-        .make_payment_and_refund(payment_method_details(), None, get_default_payment_info())
+        .make_payment_and_refund(
+            payment_method_details(100),
+            None,
```
Comment (ArjunKarthik) [inline crates/router/tests/connectors/tsys.rs:301]: pass full amount for refund

File: crates/router/tests/connectors/tsys.rs [hunk 3]
```diff
@@ -286,9 +296,14 @@ async fn should_refund_succeeded_payment_multiple_times() {
...
+            get_default_payment_info(),
+        )
         .await
         .unwrap();
+    tokio::time::sleep(Duration::from_secs(10)).await;
```
Comment (ArjunKarthik) [inline crates/router/tests/connectors/tsys.rs:306]: Do we still need this sleep time?

**Review by ArjunKarthik — CHANGES_REQUESTED** [review]

File: crates/router/tests/connectors/tsys.rs (hunk 3 above)
Comment (SamraatBansal) [reply crates/router/tests/connectors/tsys.rs:306]: Yes in this transaction is registered after a lag, that is why sleep is added

File: crates/router/tests/connectors/tsys.rs (hunk 2 above)
Comment (SamraatBansal) [reply crates/router/tests/connectors/tsys.rs:301]: It will by default pick 100, Rest at every place it is handled

**Review by ArjunKarthik — APPROVED** [review]

**Review by NishantJoshi00 — APPROVED** [review]

**Pull request merged**